pkg/nimble/autoadv/README.md

```diff
@@ -26,3 +26,18 @@ To specify a device name add the following line to your Makefile:
 ```
 CFLAGS += -DNIMBLE_AUTOADV_DEVICE_NAME='"Riot OS device"'
 ```
+
+By the default, in the advertised packet, the module includes the advertising
+**Flags** data type. According to Bluetooth Core Specification Supplement (see
+[§1.3.1](https://www.bluetooth.com/specifications/specs/core-specification-supplement-9/))
+> The Flags data type shall be included when any of the Flag bits are non-zero
+and the advertising packet is connectable, otherwise the Flags data type may be
+omitted.
+
+If your application is not connectable (eg. a temperature sensor advertising
```

Comment:
Can we determine if the application is connectable based on the modules that are used?

Reply:
I'm not sure, because although it could not be advertising, it could still connect to another peripheral device,  so I can't use `svc_gap/svc_gat` for this, and when doing info-modules I'm not sure I see anything relevant there.